src/components/Models/Model/Model.component.tsx

```diff
@@ -100,11 +100,11 @@ export const Model: FC<ModelProps> = ({
 
   const spawnComponent = useMemo(
     () => <Spawn avatar={scene} onSpawnFinish={onSpawnAnimationFinish} />,
-    [onSpawnAnimationFinish]
+    [onSpawnAnimationFinish, scene]
   );
 
   return (
-    <group ref={modelRef} dispose={null} rotation={[0, 0, 0]}>
```

Comment:
Allow react to cleanup https://gracious-keller-98ef35.netlify.app/docs/api/automatic-disposal/. Manual disposal can happen in fallback. This does need to be tested for legacy however....